electrum/tests/test_iso_639.py

```diff
@@ -0,0 +1,63 @@
+from unittest import TestCase
+
+from electrum.i18n import get_iso_639_1, languages
+
+
+class TestISO639(TestCase):
+    def test_lowercase(self):
+        self.assertEqual(
+            get_iso_639_1('EN'),
+            'en'
+        )
+        self.assertEqual(
+            get_iso_639_1('Es'),
+            'es'
+        )
+
+    def test_less_than_2_letters(self):
+        self.assertEqual(
+            get_iso_639_1(''),
+            ''
+        )
+        self.assertEqual(
+            get_iso_639_1('X'),
+            'x'
+        )
+        self.assertEqual(
+            get_iso_639_1('y'),
+            'y'
+        )
```

Comment:
I dont think we want it to work it like that. ISO-639-1 is a fixed list of language codes, not any string with <= 2 length. Its a minor issue but i would change a bit definition of that function, and test it against appropriate list.